care/facility/api/viewsets/asset.py

```diff
@@ -117,6 +157,22 @@ def set_default_user_location(self, request, *args, **kwargs):
         except:
             raise Http404
 
+    # Asset Integration API
+    @action(detail=False, methods=["POST"])
```

Comment:
use the swagger auto schema and a dummy serializer to create the swagger spec for this API

Reply:
Done. Do check the Dummy once